libraries/core_libs/consensus/src/dag/dag.cpp

```diff
@@ -288,8 +288,8 @@ DagManager::DagManager(blk_hash_t const &genesis, addr_t node_addr, std::shared_
       log_time_(log_time) {
   LOG_OBJECTS_CREATE("DAGMGR");
   if (auto ret = getLatestPivotAndTips(); ret) {
-    frontier_.pivot = std::move(ret->first);
-    for (auto const &t : ret->second) {
+    frontier_.pivot = ret->first;
```

Comment:
Why not std::move(ret->first) here?

Reply:
because some types dont use internally pointers, like form example blkh_hash_t so it does not make sense to move as nothing is gained